### ISSUE TITLE
从numpy到pytorch实现线性回归 | 朱健的个人网站

### ISSUE THREAD

**Issue by zjZSTU** [zjZSTU/guestbook#16]
https://www.zhujian.tech/posts/ca2079f0.html#more 

 参考： Linear Regression and Gradient Descent from scratch in PyTorch PyTorch进阶之路（二）：如何实现线性回归 线性回归 特征缩放 首先利用numpy实现梯度下降解决多变量线性回归问题，然后逐步将操作转换成pytorch 实现步骤如下：  加载训练数据 初始化权重 计算预测结果 计算损失函数 梯度更新 重复3-5步，直到完成迭代
